kamailio/registrar-role.cfg

```diff
@@ -73,8 +73,10 @@ route[HANDLE_REGISTER]
 
 route[ATTEMPT_AUTHORIZATION]
 {
+    route(SINGLE_CONTACT);
```

Comment:
@Dinkor since the problem is restricted to REGISTERmessages and we are in the process of changing registrar , you can just make the quick fix here and use single_contact in the lines below.

`$var(single_contact) = $(ct{s.select,0,,});`